src/content/documentation/publish/source-code-submission.md

```diff
@@ -84,6 +84,12 @@ Code submission process happens after the add-on validation step.
 
```

Comment:
I would be tempted to rewrite the section above as follows:

```
## Upload source code with your new add-on version

::: note
If you need to provide it, source code must be attached to every extension version.
:::

You provide source code after the add-on validation step during your submission.
```
I'm also going to look into whether we can provide an anchor on the step to include a link here. Also, I think it would make sense to provide a link from the [step](https://github.com/mozilla/extension-workshop/blob/3ef66969f6aaf3008636bf6c4147e2b3abf27a47/src/content/documentation/publish/submitting-an-add-on.md?plain=1#L78) to this page.